fletcher/base.py

```diff
@@ -191,53 +191,39 @@ def __init__(self, array, dtype=None, copy=None):
                     self.__class__.__name__, type(array)
                 )
             )
+        self.data.validate()
```

Comment:
there was linter error (`black --check`) before, I have fixed it (it is enough to launch `black name_of_file.py` locally), now it breaks here, as this method is not available on `ChunkedArray`

Reply:
thanks !!

Reply:
ok, it's only available starting from 0.15.